snuba/subscriptions/codecs.py

```diff
@@ -98,15 +116,20 @@ def decode(self, value: KafkaPayload) -> ScheduledSubscriptionTask:
 
         entity_key = EntityKey(scheduled_subscription_dict["entity"])
 
+        data = scheduled_subscription_dict["task"]["data"]
```

Comment:
FYI: the docstring of this class should probably change

Reply:
https://github.com/getsentry/snuba/pull/6499/commits/a642b1b997cf6697615f5406e79b94d5dbdfcf6a#diff-f48e31c677eccff7bf14b091fe348640c394684c53c8d3a5a16d27e265d8b1abR87